src/App.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import {BrowserRouter as Router, Switch, Route} from 'react-router-dom'
 import './App.css';
-import Header from './components/header/header'
+import footer from './components/Footer/footer';
 
 function App() {
   return (
     <div>
       <Router>
       <Switch>
-      <Route path="/" exact component={Header}/>
+      <Route path="/" exact component={footer}/>
       </Switch>
       </Router>
```

Comment:
Its not the correct. 
1) Routing is all about  different pages! 
2) Footer Component will come below the outside Switch Tag! 
3) For more details, Refer how to do routing in this video! https://www.youtube.com/watch?v=Law7wfdg_ls&t=1778s
4) F for footer will be capitalized

Reply:
I have corrected 
1. F capital in footer component.
2. Outside switch tag added route path with /Footer so is this correct ?

Reply:
@smilegupta

Reply:
Hey @smilegupta I changed the code but am stuck in the pr. I dont know why its having every new pr while commiting the code.

Reply:
@karshil2309 what exactly are facing issue!? Can you brief more about it.

Reply:
@smilegupta I have done the changes but I did some blunders with git. So may be my code hasnt been updated. Can I do new PR and update it ?